tests/installation/partitioning_raid.pm

```diff
@@ -42,7 +42,12 @@ sub addpart {
     send_key $cmd{donotformat};
     send_key "tab";
 
-    send_key_until_needlematch 'partition-selected-raid-type', 'down';
+    if ($size == 300 and get_var('UEFI')) {
```

Comment:
That is not at all self-explanatory. Please be a bit more explicit, e.g. extract 300 as a constant with good name and potentially also a comment describing the need.